flash_patcher/parse/visitor/stage_visitor.py

```diff
@@ -59,9 +60,13 @@ def visitPythonFile(
         example output: "DoAction1.as,DoAction2.as"
         Python script names may not include spaces.
         """
+        cwd = Path.cwd()
```

Comment:
Would it be possible to move this into a helper function in the `util` folder? If so we should do that, like how we did with the safe file handling.

Reply:
Done!